contracts/reverseRegistrar/ISignatureReverseResolver.sol

```diff
@@ -0,0 +1,36 @@
+pragma solidity >=0.8.4;
+
+interface ISignatureReverseResolver {
+    event VersionChanged(bytes32 indexed node, uint64 newVersion);
+    event ReverseClaimed(address indexed addr, bytes32 indexed node);
+    event NameChanged(bytes32 indexed node, string name);
```

Comment:
These events too.

Reply:
They are all called within `SignatureReverseResolver` so can't be moved